ui/components/component-library/button-link/button-link.stories.js

```diff
@@ -68,6 +68,9 @@ export default {
     href: {
       control: 'text',
     },
+    externalLink: {
+      control: 'boolean',
+    },
     startIconName: {
       control: 'select',
       options: Object.values(ICON_NAMES),
```

Comment:
I know we are duplicating some code here but I think it's worth adding the ExternalLink story to ButtonLink as well

```
export const ExternalLink = (args) => (
  <ButtonLink {...args}>Anchor element with external link</ButtonLink>
);

ExternalLink.args = {
  href: 'https://metamask.io',
  externalLink: true,
};
```

Reply:
yup!